js/app.js

```diff
@@ -274,7 +245,12 @@ function App() {
 				modules.push(module.module);
 			}
 		}
-		await loadModules(modules);
+
+		Log.log("Loading module helpers ...");
+		modules.forEach((module) => {
+			loadModule(module);
+		});
+		Log.log("All module helpers loaded.");
```

Comment:
greatly simplified this, it shouldnt break exisiting module loading at least for me.